scss/tech.scss

```diff
@@ -1,42 +1,25 @@
 $tech-logo-size: 100px;
 
 .techs-container {
-  display: flex;
-  flex-wrap: wrap;
-  margin-left: 100px;
+  display: grid;
+  margin: 0 auto;
+  width: 900px;
+  grid-template-columns: repeat(3, 300px);
+  grid-template-rows: 1fr 1fr;
+  grid-row-gap: 50px;
 }
 
-.tech{
+.tech,
+.tech-col {
```

Comment:
Adding this 2 is not needed, keep one and add another class to set the technology as the primary so it will be placed as a column